docs/use/dapp-staking/for-stakers/staking.md

```diff
@@ -0,0 +1,73 @@
+---
+sidebar_position: 1
+---
+
+import Figure from "/src/components/figure"
+
+# How to stake
+
+:::warning
+
+ **To earn rewards continuously, users have to vote and stake every new period.**
+
+ :::
+
+At the end of a Period, tokens are unstaked from all dApps, remain locked on the account but must be staked back on dApps for the new Period.
+
+### Visit dApp Staking on the Portal
+
+Go to the [Astar Portal](https://portal.astar.network/astar/dapp-staking/discover):  and click on the `dApp Staking` page.
+
+Make sure you are connected to the correct network.
+- **Shibuya Network**: testnet
+- **Shiden Network**: mainnet on Kusama
+- **Astar Network**: mainnet on Polkadot
+
+<div style={{textAlign: 'center'}}>
+
+![1](img/1.png)
+</div>
+
+### Discover Projects
+
+:::tip
+
+Staked TVL and Project Leaderboards are available at the top of the page. 
+
+
+:::
+
+Click on the project cards for more information, access to their communities or to view team accounts. You can also find useful resources on our [forum](https://forum.astar.network/) or check [DefilLama](https://defillama.com/) or [DappRadar](https://dappradar.com/).
+
+<Figure src={require('/docs/use/dapp-staking/for-stakers/img/Projects_page.png').default } width="75%" /> 
+
+### Vote & Stake
+
+1) On the [dApp staking](https://portal.astar.network/astar/dapp-staking/discover) page, you can either:
+
+- Click on the `Vote/Stake today` button;
+- Select your favorite project and click on the `Vote & Stake` button on the project page;
+
+<Figure src={require('/docs/use/dapp-staking/for-stakers/img/Vote_Page.png').default } width="75%" /> 
+
+2) Once on the voting page, you can choose as many projects as you would like to support;
+
+<Figure src={require('/docs/use/dapp-staking/for-stakers/img/Staking_Page.png').default } width="75%" /> 
+
+3) To stake, enter the desired amount of tokens for the dApp(s) you've chosen. This may be the same or a different amount for each.
+
+:::tip
+
+Make sure to respect the minimum amount to stake and always keep some tokens free for gas to manage the claiming and re-staking transactions.. The minimum amount to stake per dApp is 50 SDN/500 ASTR.
```

Comment:
This is not up to users to respect, any operation that would result with less than limit being locked/staked will fail.

Reply:
Fixed